### PR TITLE
Updated python SDK to pick up new powerState field

### PR DESCRIPTION
**Pull request by PakDLiu**

### CODE REVIEW

**Comment by PakDLiu** [comment]
All these changes other than the `setup.py `are generated by the SDK.

Built the extension and added it to my own az cli.

Saw the new field:
```
pak@pak-ubuntu-dev:~/src/azure-hanaonazure-cli-extension$ az hanainstance show --resource-group SJCT-T050 --instance-name azlinux0033
{
  "hanaInstanceId": "609ec680-ef20-4231-b327-bc674ddf7308",
  "hardwareProfile": {
    "hanaInstanceSize": "S72m",
    "hardwareType": "Cisco_UCS"
  },
  "id": "/subscriptions/b7079faf-f1ba-42fb-97e8-7fcf3b2f7cb3/resourceGroups/SJCT-T050/providers/Microsoft.HanaOnAzure/hanaInstances/azlinux0033",
  "location": "centralus",
  "name": "azlinux0033",
  "networkProfile": {
    "circuitId": "/subscriptions/e6806823-d4a6-4ccf-b8a3-656e96993c59/resourceGroups/T050_AZlinux-USW-Circuits/providers/Microsoft.Network/expressRouteCircuits/T050_AZlinux-USW01-5GB",
    "networkInterfaces": [
      {
        "ipAddress": "172.16.34.33"
      }
    ]
  },
  "osProfile": {
    "computerName": "azlinux0033",
    "osType": "SLES 12 SP2",
    "version": "12 SP2"
  },
  "powerState": null,
  "resourceGroup": "SJCT-T050",
  "storageProfile": {
    "nfsIpAddress": "10.250.51.53",
    "osDisk": [
      {
        "diskSizeGB": "20",
        "name": "FakeOSVolume0"
      }
    ],
    "osDisks": null
  },
  "tags": {},
  "type": "Microsoft.HanaOnAzure/hanaInstances"
}

```

File: setup.py [hunk 1]
```diff
@@ -26,7 +26,7 @@
 
 DEPENDENCIES = [
     'msrestazure',
-    'azure-mgmt-hanaonazure==0.1.1'
+    'azure-mgmt-hanaonazure==0.2.0'
```
Comment (lagalbra) [inline setup.py:29]: Why is this changing to "2.0", but line 11 is changing from "1.1" to "1.2"?

**Review by lagalbra — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (PakDLiu) [reply setup.py:29]: 0.1.2 is the new extension version
2.0 is the new SDK version

Comment (lagalbra) [reply setup.py:29]: Oh, weird. So the SDK version is out of our hands, yes? Should we just change the "1.2" version to "2.0" to match? Is that under our control?

Comment (PakDLiu) [reply setup.py:29]: The SDK version is out of our control
https://pypi.org/project/azure-mgmt-hanaonazure/0.2.0/

We could change our version to whatever we want, I can change it to 2.0 to match the SDK version if you want

Comment (lagalbra) [reply setup.py:29]: Unless you have any objections, I think this is the best option for us. We have to bump the version to something, and as someone who has correlated version numbers between 3 different packages in a previous project, it's not worth it.

**Comment by lagalbra** [comment]
Don't you have to re-generate every other file again?

EDIT: I was mistaken; setup.py change was not made before generation of other files

**Review by lagalbra — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (jovillas) [reply setup.py:29]: wait what? no!
We don't own the SDK, it is a dependency, we could very well have bumped the extension version without changing the SDK one.
Now, if we want to change the extension tomorrow (and bump version) and the SDK haven't released this week, are we forced to keep 2.0 to keep on par with the SDK? should we bump to 2.1?  I think we should decouple those versions.
*strongly opposed*

Comment (lagalbra) [reply setup.py:29]: Jorge has a point here; I would be fine if you kept it as a bump to 0.1.2.

**Review by jovillas — APPROVED** [review]

**Pull request merged**